### ISSUE TITLE
When deleting one before the last message in the outlook express message list NVDA fails to announce new item in focus

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#446]
**Reported by pvagner on 2009-10-29 19:14**
STR:
1) Open your outlook express and focus the list of messages.
2) Make sure focus is set to the item immediately before the last item in the list (the second item from the very bottom).
3) Press the delete key.

Actual results:
NVDA stays silent and you need to use NVDA+tab or NVDA+up arrow to get the currently selected item spoken.

Expected results:
As the last message in the list gains the focus, NVDA should correctly announce to that fact.

**Comment by leonardder** [comment]
I'd like to suggest closing this following @jcsteh's rationale in #508 

> There's no point in us working on additional support for something which is now obsolete.

@JCSTeh: How about adding an obsolete label for those tickets or something to identify them?

**Comment by jcsteh** [comment]
Agreed.

[**@leonardder**](https://github.com/leonardder) commented on [9 Jun 2017, 04:44 GMT+10](https://github.com/nvaccess/nvda/issues/446#issuecomment-307192426 "2017-06-08T18:44:19Z - Replied by Github Reply Comments"):
> @JCSTeh: How about adding an obsolete label for those tickets or something to identify them?

Thanks for the suggestion, but I don't think this is necessary. This is not overly common and I don't think it's something we ever need to search by, gather statistics on, etc.

Closing as won't fix.
